.pre-commit-config.yaml

```diff
@@ -18,17 +18,3 @@ repos:
         language: system
         types: [ python ]
         stages: [ commit, push ]
-      - id: mypy
```

Comment:
Sorry about this, but I couldn't bother with mypy just for opening a d Draft.

Reply:
It's a good point actually, maybe we should disable this for draft PRs